be/src/storage/lake/update_manager.cpp

```diff
@@ -415,7 +415,7 @@ Status UpdateManager::get_column_values(Tablet* tablet, const TabletMetadata& me
                                          const TabletSchemaCSPtr& tablet_schema,
                                          const std::vector<uint32_t>& rowids) -> Status {
         std::string path = tablet->segment_location(segment_name);
-        auto segment = Segment::open(fs, path, segment_id, tablet_schema);
+        auto segment = Segment::open(fs, FileInfo{path}, segment_id, tablet_schema);
```

Comment:
Pass the `FileInfo` from the outer layer and populate its `size` field, if possible.

Reply:
Should change line 430 below to open `RandomAccessFile` with `FileInfo`